core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -18,36 +18,32 @@
         background-size: cover;
       }
       .exploration-list-table {
-        top: 168px;
-        left: 10%;
-        max-width: 675px;
-        margin-top: 20px;
-        padding-bottom: 25px;
-        padding-left: 20px;
-        padding-right: 20px;
-        position: fixed;
         background: #fff;
         border-radius: 2px;
         box-shadow: 0 2px 4px rgba(0, 0, 0, 0.14), 0 2px 4px rgba(0, 0, 0, 0.23);
-        padding-top: 20px;
-        text-align: left;
         float: left;
+        left: 10%;
+        margin-top: 20px;
+        max-width: 675px;
+        padding: 20px 20px 25px 20px;
+        position: fixed;
+        text-align: left;
+        top: 168px;
       }
       .card-preview-panel {
-        top: 188px;
-        right: 70px;
-        padding-top: 30px;
-        padding-bottom: 70px;
-        padding-left: 75px;
-        padding-right: 75px;
-        position: fixed;
         background: #fff;
         border-radius: 2px;
         box-shadow: 0 2px 4px rgba(0, 0, 0, 0.14), 0 2px 4px rgba(0, 0, 0, 0.23);
-        text-align: left;
+        padding: 30px 75px 70px 75px;
         pointer-events: none;
+        position: fixed;
+        right: 70px;
+        text-align: left;
+        top: 188px;
+      }
+      td {
```

Comment:
This is too broad, and will apply to every td in the app. Can we use a more specific css selector instead (and also get rid of !important)?